src/pages/Main/Main.jsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import theme from 'styles/theme';
 import styled from 'styled-components';
+import Logo from 'components/Logo/Logo';
 
 function Main(props) {
-  return <TestBox>여긴 메인페이지다.</TestBox>;
+  return (
+    <>
+      <Logo fontSize="30px" />
+      <TestBox>여긴 메인페이지다.</TestBox>
+    </>
+  );
 }
 
 const TestBox = styled.div`
-  color: ${theme.colors.red};
+  color: ${theme.red};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.red};
```
import 하지 않고 사용할 수 있습니다